package.json

```diff
@@ -3,39 +3,60 @@
   "description": "Properly's backend logger for JavaScript",
   "license": "UNLICENSED",
   "author": "Properly Inc",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/GoProperly/js-proper-logger.git"
-  },
   "version": "0.0.1",
-  "main": "index.js",
-  "files": [],
```

Comment:
I think we'll want to keep the [`files`](https://docs.npmjs.com/files/package.json#files) config (e.g. `"files": ["dist/*"]`) to avoid NPM packaging all the config files/source code in the output package. I think there is also a way to do that using `.npmignore` but I have not used that previously.